packages/allure-codeceptjs/README.md

```diff
@@ -6,21 +6,15 @@
 npm i -D allure-codeceptjs
 ```
 
```

Comment:
i guess we also need note like this "Add the allure plugin inside you plugins section of `codecept.config.js`"